lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -1,5 +1,17 @@
 # feature
 ---
+fabric:
+  _exclude: [/N3/]
+  config_get: "show feature-set"
+  config_get_token: '/^fabric[\s\d]+(\w+)/'
+  config_set: "<state> feature-set fabric"
```

Comment:
`<state>` is required here because the process of enabling the `fabric` feature-set is:

```
install feature-set fabric
feature-set fabric
```

If `install` was hardcoded, the feature-set wouldn't be enabled.